lxd/daemon.go

```diff
@@ -55,7 +55,8 @@ type Daemon struct {
 	shutdownChan chan struct{}
 
 	// Tasks registry for long-running background tasks.
-	tasks task.Group
+	tasks        task.Group
+	clusterTasks task.Group
```

Comment:
I would mention in the comment above that the reason we keep cluster tasks separate is to save on power consumption.

Reply:
updated